### PR TITLE
Svelte setup

### PR DESCRIPTION
**Pull request by jakeanq**
* Added Svelte
* Changed webpack to SvelteKit (Vite) since it was recommended by the Svelte setup guide
* Added Storybook

### CODE REVIEW

File: src/lib/Animation.svelte [hunk 1]
```diff
@@ -1,11 +1,26 @@
-@import "./Assets/Fonts/Lato.scss";
+<script lang="ts">
+    import Logo from "./assets/Images/EM.svg?component";
+</script>
+
+<div class="background">
+    <div class="em-test">
+        <div>
+            <Logo class="sweep" />
```
Comment (RVeips) [inline src/lib/Animation.svelte]: Replace with
`<Logo /><Logo class="sweep" />`
for the perfect animation layout (gray background image below sweep animation)

**Comment by RVeips** [comment]
@jakeanq in the `Project Structure` section of the README - maybe it would be useful add some info on how the svelte project is set up? What UI files/functionality are found in which folders.

**Comment by jakeanq** [comment]
@RVeips I've updated the README; I'm still getting my head around Svelte and CEF etc so it's a bit vague still...  Can you think of anything I've missed?

**Comment by RVeips** [comment]
@jakeanq All the important stuff seems to be there. (Maybe the only small thing I would change is rename "lib" to "components". "lib" might be a bit confusing)

**Comment by jakeanq** [comment]
For the `lib` vs `components`, I went based on the example structure I found here: https://kit.svelte.dev/docs/project-structure

Also I think you can reference that folder using `$lib`.  Perhaps `lib/components` would make sense?  Or we could add sub-folders for different types of components.

Also @ryesearchanddevelopment thoughts? I think you said you'd done some Svelte stuff.

**Comment by RVeips** [comment]
@jakeanq Right, example structures are usually the most common, so leave it as `lib` in case we need to add something that is not a component to it.

In that case I think `lib/components` is a good option.

**Comment by ryesearchanddevelopment** [comment]
Yes, lib is a special directory that is accessed by $lib. I put shared UI components in lib/components/ui.

**Comment by RVeips** [comment]
@ryesearchanddevelopment would there be other types of components? To make a folder called "ui" implies that there might be non-ui components

**Comment by ryesearchanddevelopment** [comment]
You can make components that aren't ui, I think that's pretty rare though. Definitely rare enough that just having a components folder would be fine.

**Comment by jakeanq** [comment]
`$lib/components` sounds good to me, updated

**Review by RVeips — APPROVED** [review]

**Pull request merged**